cpg_workflows/stages/seqr_loader_long_read/long_read_snps_indels_annotation.py

```diff
@@ -0,0 +1,428 @@
+
+        # guessing at resource requirements
+        merge_job.cpu(4)
+        merge_job.memory('16Gi')
+        merge_job.storage('50Gi')
+        merge_job.declare_resource_group(output={'vcf.bgz': '{root}.vcf.bgz', 'vcf.bgz.tbi': '{root}.vcf.bgz.tbi'})
+
+        # option breakdown:
+        # -Oz: bgzip output
+        # -o: output file
+        # --threads: number of threads to use
+        # -m: merge strategy
+        # -0: compression level
+        merge_job.command(
+            f'bcftools merge {" ".join(batch_vcfs)} -Oz -o '
+            f'{merge_job.output["vcf.bgz"]} --threads 4 -m none -0',  # type: ignore
+        )
+        merge_job.command(f'tabix {merge_job.output["vcf.bgz"]}')  # type: ignore
+
+        # write the result out
+        get_batch().write_output(merge_job.output, str(outputs['vcf']).removesuffix('.vcf.bgz'))  # type: ignore
+
+        return self.make_outputs(multicohort, data=outputs, jobs=merge_job)
+
+
+@stage(required_stages=MergeLongReadSNPsIndels)
+class SiteOnlyVCFs(MultiCohortStage):
+    """
+    Get the site-only VCFs from the merged VCF
+    """
+
+    def expected_outputs(self, multicohort: MultiCohort) -> dict:
+        """
+        Generate site-only VCFs from the merged VCF.
+        """
+        return {
+            # 'siteonly': to_path(self.prefix / 'siteonly.vcf.gz'),
+            'siteonly_part_pattern': str(self.prefix / 'siteonly_parts' / 'part{idx}.vcf.gz'),
+        }
+
+    def queue_jobs(self, multicohort: MultiCohort, inputs: StageInput) -> StageOutput | None:
+        """
+        Submit jobs.
+        """
+        jobs = []
+        outputs = self.expected_outputs(multicohort)
+        scatter_count = joint_calling_scatter_count(len(multicohort.get_sequencing_groups()))
+        out_siteonly_vcf_part_paths = [
+            to_path(outputs['siteonly_part_pattern'].format(idx=idx)) for idx in range(scatter_count)
+        ]
+
+        intervals_path = None
+        if config_retrieve(['workflow', 'intervals_path'], default=None):
+            intervals_path = to_path(config_retrieve(['workflow', 'intervals_path']))
+
+        exclude_intervals_path = None
+        if config_retrieve(['workflow', 'exclude_intervals_path'], default=None):
+            exclude_intervals_path = to_path(config_retrieve(['workflow', 'exclude_intervals_path']))
+
+        vcf_jobs = split_merged_vcf_and_get_sitesonly_vcfs_for_vep(
+            b=get_batch(),
+            scatter_count=scatter_count,
+            merged_vcf_path=inputs.as_path(multicohort, MergeLongReadSNPsIndels, 'vcf'),
+            tmp_bucket=self.tmp_prefix / 'tmp',
+            # out_siteonly_vcf_path=outputs['siteonly'],
+            out_siteonly_vcf_part_paths=out_siteonly_vcf_part_paths,
+            intervals_path=intervals_path,
+            exclude_intervals_path=exclude_intervals_path,
+            job_attrs=self.get_job_attrs(),
+        )
+        jobs.extend(vcf_jobs)
+        for job in jobs:
+            assert job
```

Comment:
I'm not sure what this assert is doing - the method above already grooms out `None` jobs from the list of jobs, so it returns an empty list or a list of Jobs. Probably best to drop it, or add a statement to make it informative if it fails